gui/elements/_window.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, viewer, show=True):
 
         self.viewer.events.status.connect(self._status_changed)
         self.viewer.events.help.connect(self._help_changed)
+        self.viewer.window = self
```

Comment:
self.viewer.window.viewer.window... :P

Reply:
:-( indeed, less than ideal, but the only way i can see how to do it. Also this is possible in matplotlib, but doesn't seem to cause them problems (as far as i know)

Reply:
this is now fixed, thanks to @kne42!